components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +267,35 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
```

Comment:
ahh ok. unrelated to this pr but curious what happens if a rustfmt changes whitespace in the sql @raskchanky ?